.travis.yml

```diff
@@ -4,9 +4,12 @@ sudo: false
 
 matrix:
   include:
-    # Only one test for these Python versions
+    # One build with minimum versions of dependencies
     - env: ENV_FILE="ci/travis/35-minimal.yaml"
 
+    # one build with no optional dependencies
+    - env: ENV_FILE="ci/travis/38-no-optional-deps.yaml" PYGEOS=true
```

Comment:
@jorisvandenbossche is `pygeos` being considered a hard requirement in this context?  It seems like perhaps that should still be kept as optional for purposes of this PR?

Reply:
Yeah, I was thinking that also for when using pygeos, we need to ensure the other optional ones are not used. But you're right that now testing that pygeos is actually optional is probably more useful